frontend/src/metabase/query_builder/components/filters/modals/InlineValuePicker/InlineValuePicker.styled.ts

```diff
@@ -7,8 +7,13 @@ import { color } from "metabase/lib/colors";
 
 import NumericInput from "metabase/core/components/NumericInput";
 
-export const ValuesPickerContainer = styled.div`
-  ul.input {
```

Comment:
this input classname got removed somewhere along the way, and we want to add a custom focus style here